File: Iniciante/1065 - Pares entre Cinco Números/Python/1065.py
def main():
    pairs = check_numbers()
    print(f"{pairs} valores pares")


def check_numbers():
    even_number_counter = 0
    for _ in range(5):
        number = int(input())
        if number % 2 == 0:
            even_number_counter += 1
    return even_number_counter


if __name__ == "__main__":
    main()

File: Iniciante/1065 - Pares entre Cinco Números/TypeScript/1065.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.split("\n");
  const pairs: number = checkNumbers(lines);
  console.log(`${pairs} valores pares`);
}

function checkNumbers(array: string[]): number {
  let evenNumberCounter: number = 0;
  for (let i = 0; i < 5; i++) {
    let number: number = Number(array[i]);
    if (number % 2 === 0) {
      evenNumberCounter++;
    }
  }
  return evenNumberCounter;
}

main();
